contracts/strategies/ConverterStrategyBase.sol

```diff
@@ -339,12 +324,6 @@ abstract contract ConverterStrategyBase is ITetuConverterCallback, DepositorBase
     uint[] memory withdrawnAmounts = _depositorEmergencyExit();
     emit OnDepositorEmergencyExit(withdrawnAmounts);
 
-    (address[] memory tokens, uint indexAsset) = _getTokens(asset);
-
-    // convert amounts to main asset and update base amounts
-    (uint collateral, uint[] memory repaid) = _convertAfterWithdrawAll(tokens, indexAsset);
```

Comment:
there is same problem here. After emergency exist we should have only underlying on the balance, conversion is necessary